web/src/users/login.rs

```diff
@@ -0,0 +1,110 @@
+use axum::{
+    extract::State,
+    response::{IntoResponse, Redirect},
+    Form,
+};
+use cja::{
+    app_state::AppState as _,
+    tower_cookies::{Cookie, Cookies},
+};
+
+use db::users::User;
+
+use password_auth::verify_password;
+
+use crate::{sessions::Session, templates::Template, AppState, WebResult};
+
+pub async fn get(t: Template) -> impl IntoResponse {
+    t.render(maud::html! {
+        form method="post" action="/login" {
+            input type="text" name="username" placeholder="Username";
+            input type="password" name="password" placeholder="Password";
+            input type="submit" value="Login";
+        }
+    })
+}
```

Comment:
Here is where we build the login page!

Its a very boring HTML form for now lol